static/app/utils/profiling/flamegraph.ts

```diff
@@ -346,14 +347,16 @@ export class Flamegraph {
   }
 
   findAllMatchingFrames(frameName?: string, framePackage?: string): FlamegraphFrame[] {
+    framePackage = tryTrimPackage(framePackage);
+
     const matches: FlamegraphFrame[] = [];
 
     for (let i = 0; i < this.frames.length; i++) {
       if (
         this.frames[i].frame.name === frameName &&
         // the framePackage can match either the package or the module
         // this is an artifact of how we previously used image
-        (this.frames[i].frame.package === framePackage ||
+        (tryTrimPackage(this.frames[i].frame.package) === framePackage ||
```

Comment:
Dang, yet another thing that can fall out of sync 😢  Is there any documentation of why we trim this btw and do we care about trimming since this is just for search/highlighting anyways?

Reply:
So to align with how other events, we're now storing the full package in the profile. The full package value looks like an absolute path leading to a `.so/.dylib/etc`. This absolute path can vary depending on which machine it was built on and for aggregation purposes (especially across multiple releases), we need a stable package to be able to aggregate on. The `trimPackage` function accomplishes that by trimming the package and producing a stable package name that we can use. The backend will also implement this so we may be able to eventually remove this from the frontend but something to double check later.

Reply:
>The backend will also implement this so we may be able to eventually remove this from the frontend but something to double check later.

Nice, yeah makes sense. Would be nice for everything downstream to just benefit from this ❤️